### ISSUE TITLE
Animal Crossing - Crashing on Start

### ISSUE THREAD

**Issue by codbugpatrol** [Ryujinx/Ryujinx#2453]
### What's the issue you encountered?
While starting the Animal Crossing New Horizons it crash right at the begging. 

### How can the issue be reproduced?
Try to start the game and soon it will crash.

![AnimalCrossing_Crash](https://user-images.githubusercontent.com/85196112/124981285-c3d96b00-e00b-11eb-809a-779e5d4668fc.jpg)

### Log file

```
00:00:39.759 |E| GUI.RenderLoop Gpu GLERROR: DebugSeverityHigh: GL_INVALID_VALUE error generated. <count> must not be negative.
CallStack=   at System.Environment.get_StackTrace()
   at Ryujinx.Graphics.OpenGL.Debugger.GLDebugHandler(DebugSource source, DebugType type, Int32 id, DebugSeverity severity, Int32 length, IntPtr message, IntPtr userParam)
   at Ryujinx.Graphics.OpenGL.Pipeline.DrawIndexedImpl(Int32 indexCount, Int32 instanceCount, IntPtr indexBaseOffset, Int32 firstVertex, Int32 firstInstance)
   at Ryujinx.Graphics.OpenGL.Pipeline.DrawIndexed(Int32 indexCount, Int32 instanceCount, Int32 firstIndex, Int32 firstVertex, Int32 firstInstance)
   at Ryujinx.Graphics.Gpu.Engine.Methods.DrawEnd(GpuState state, Int32 firstIndex, Int32 indexCount)
   at Ryujinx.Graphics.Gpu.Engine.Methods.DrawEnd(GpuState state, Int32 argument)
   at Macro(MacroJitContext , GpuState , Int32 )
   at Ryujinx.Graphics.Gpu.Engine.GPFifo.GPFifoProcessor.Send(MethodParams meth)
   at Ryujinx.Graphics.Gpu.Engine.GPFifo.GPFifoProcessor.Process(ReadOnlySpan`1 commandBuffer)
   at Ryujinx.Graphics.Gpu.Engine.GPFifo.GPFifoDevice.DispatchCalls()
   at Ryujinx.Ui.RendererWidgetBase.Render()
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()

00:00:39.767 |E| GUI.RenderLoop Application : Unhandled exception caught: Ryujinx.Memory.InvalidMemoryRegionException: Not mapped: va=0xFFFFFFFFFFFFFFFF, size=0x0000000000000004
   at Ryujinx.Cpu.MemoryManagerHostMapped.AssertMapped(UInt64 va, UInt64 size)
   at Ryujinx.Cpu.MemoryManagerHostMapped.GetSpan(UInt64 va, Int32 size, Boolean tracked)
   at Ryujinx.Graphics.Gpu.Memory.MemoryManager.ReadImpl(UInt64 va, Span`1 data, Boolean tracked)
   at Ryujinx.Graphics.Gpu.Memory.MemoryManager.Read[T](UInt64 va)
   at Ryujinx.Graphics.Shader.Translation.ShaderHeader..ctor(IGpuAccessor gpuAccessor, UInt64 address)
   at Ryujinx.Graphics.Shader.Translation.Translator.DecodeShader(UInt64 address, IGpuAccessor gpuAccessor, TranslationOptions options, TranslationCounts counts, ShaderConfig& config)
   at Ryujinx.Graphics.Shader.Translation.Translator.CreateContext(UInt64 address, IGpuAccessor gpuAccessor, TranslationOptions options, TranslationCounts counts)
   at Ryujinx.Graphics.Gpu.Shader.ShaderCache.DecodeGraphicsShader(GpuState state, TranslationCounts counts, TranslationFlags flags, ShaderStage stage, UInt64 gpuVa)
   at Ryujinx.Graphics.Gpu.Shader.ShaderCache.GetGraphicsShader(GpuState state, ShaderAddresses addresses)
   at Ryujinx.Graphics.Gpu.Engine.Methods.UpdateShaderState(GpuState state)
   at Ryujinx.Graphics.Gpu.Engine.Methods.UpdateState(GpuState state, Int32 firstIndex, Int32 indexCount)
   at Ryujinx.Graphics.Gpu.Engine.Methods.DrawEnd(GpuState state, Int32 firstIndex, Int32 indexCount)
   at Ryujinx.Graphics.Gpu.Engine.Methods.DrawEnd(GpuState state, Int32 argument)
   at Macro(MacroJitContext , GpuState , Int32 )
   at Ryujinx.Graphics.Gpu.Engine.GPFifo.GPFifoProcessor.Send(MethodParams meth)
   at Ryujinx.Graphics.Gpu.Engine.GPFifo.GPFifoProcessor.Process(ReadOnlySpan`1 commandBuffer)
   at Ryujinx.Graphics.Gpu.Engine.GPFifo.GPFifoDevice.DispatchCalls()
   at Ryujinx.Ui.RendererWidgetBase.Render()
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)

--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
```

### Environment?

- Ryujinx version: 1.0.6956</br>
- Game version: 1.10.0</br>

- System Specs:
  - OS: Windows 10
  - CPU: Ryzen 5 3600
  - GPU:  NVIDIA GTX 670
  - RAM: 16GB
  
- Applied Mods : Yes
All DLCs applied.

### Additional context?
Using Firmware Version: 12.0.3

[AnimalCrossing_Crash_Ryujinx_1.0.6956.log](https://github.com/Ryujinx/Ryujinx/files/6786889/AnimalCrossing_Crash_Ryujinx_1.0.6956.log)

**Comment by Leischii** [comment]
This is a known problem and is noted in the compatibility entry of the game. use a save game to skip this part or try until it doesn't crash.

Issue can be closed again.

**Comment by ryulovesjinx** [comment]
> This is a known problem and is noted in the compatibility entry of the game. use a save game to skip this part or try until it doesn't crash.
> 
> Issue can be closed again.

I understand your point of view, but this is an issue should remain opened until the bug is closed. I did not find that information elsewhere to know that this is a known issue. This should be closed if the there is another bug with the same problem.

**Comment by Sinigon** [comment]
Try to create a new profile im options of Ryujinx (it will create a new prof with a "normal" id). It helped me.

**Comment by Zenkibou** [comment]
> Try to create a new profile in options of Ryujinx (it will create a new prof with a "normal" id). It helped me.

Indeed it seems to fix the crash for me as well.

**Comment by sakuka47** [comment]
Create a first backup with the 1.4 maj then after you can play in 2.0.5

**Comment by MetrosexualGarbodor** [comment]
Fixed by #3741.
